deployer/cluster.py

```diff
@@ -73,36 +73,17 @@ def deploy_support(self):
         cert_manager_url = "https://charts.jetstack.io"
         cert_manager_version = "v1.3.1"
 
-        print_colour("Adding cert-manager chart repo...")
-        subprocess.check_call(
-            [
-                "helm",
-                "repo",
-                "add",
-                "jetstack",
-                cert_manager_url,
-            ]
-        )
-
-        print_colour("Updating cert-manager chart repo...")
-        subprocess.check_call(
-            [
-                "helm",
-                "repo",
-                "update",
-            ]
-        )
-
         print_colour("Provisioning cert-manager...")
         subprocess.check_call(
             [
                 "helm",
                 "upgrade",
+                "cert-manager",  # given release name (aka. installation name)
+                "cert-manager",  # helm chart to install
```

Comment:
I thought if we don't explicitly add the repo and specify where it comes from, it comes from the old deprecated kubernetes/charts repo, and we'll get https://github.com/helm/charts/tree/master/stable/cert-manager

Reply:
The line below, `f"--repo={cert_manager_url}",`, is added to explicitly make us rely on the helm chart repo specified with `--repo`, and search for a chart named `cert-manager` in that.

Reply:
I just somehow completely missed that!